pkgs/applications/science/biology/slicer3d/default.nix

```diff
@@ -0,0 +1,110 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+
+  desktopItem = makeDesktopItem {
+    name = "Slicer3D";
+    exec = "SlicerApp-real";
+    desktopName = "Slicer 3D";
+    genericName = "A 3D Slicer Program"; # edit later
+    categories = [ "Graphics" ]; # Edit later
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version;
+  src = fetchzip {
+    name = "Slicer-${version}-linux-amd64.tar.gz";
+    url = "http://download.slicer.org/download?os=linux&stability=release&version=${version}";
+    hash = "sha256-DmJS1yrwJBcAIRnVA8VdsO4u82E1MWX+uvRC+6dEXmM=";
+    extension = "tar.gz";
+  };
+
+  desktopItems = [
+    desktopItem
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    freetype
+    xorg.libSM
+    xorg.libICE
+    xorg.libXrender
+    xorg.libXext
+    xorg.libX11
+    zlib
+    stdenv.cc.cc.lib
+    libGL
+    libGLU
+    fontconfig
+    xorg.xkbutils
+    xorg.xcbutil
+    xorg.xcbutilwm
+    xorg.xcbutilimage
+    xorg.xcbutilrenderutil
+    xorg.xcbutilkeysyms
+    xorg.libXdamage
+    xorg.libXfixes
+    xorg.libXrandr
+    xorg.libXcursor
+    xorg.libXtst
+    xorg.libXcomposite
+    alsa-lib
+    postgresql
+    libxkbcommon
+    glib
+    pulseaudio
+    cups
+    unixODBC
+    libxcrypt-legacy
+    nss
+    nspr
+  ];
+
+  autoPatchelfIgnoreMissingDeps = [ "libhwloc.so.5" ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir $out
+    cp -r . $out
+    mkdir $out/share/applications/
+    cp ${desktopItem}/share/applications/* $out/share/applications/
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Multi-platform, free open source software for visualization and image computing.";
+    sourceProvenance = sourceTypes.binaryNativeCode;
+    mainProgram = "Slicer3D";
+    homepage = "https://www.slicer.org";
+    license = licenses.bsd1;
+    platforms = lib.platforms.linux;
```

Comment:
nit: also seems there is a changelog for minor releases (though missing some such as 5.4, but has 5.6, 5.2, 5.0) at

```
https://discourse.slicer.org/t/slicer-${lib.versions.major ...}-${lib.versions.minor ...}-summary-highlights-and-changelog
```
you might consider adding